webpack.config.js

```diff
@@ -31,6 +31,7 @@ const resolve = {
     FilterForm: path.resolve(__dirname, './jsx/FilterForm'),
     Form: path.resolve(__dirname, './jsx/Form'),
     Loader: path.resolve(__dirname, './jsx/Loader'),
+    LoadingBar: path.resolve(__dirname, './jsx/LoadingBar'),
```

Comment:
@HenriRabalais any chance we can remove this from here? why is it needed in the webpack ?